docs: index and customization pages (#359)
scaleway/scaleway-ui: src/__stories__/Customization.stories.mdx

import { Meta } from '@storybook/addon-docs/blocks'
import { Box } from '..'
import { colors } from '../theme'

> <Meta title="Customization" />

Reviewer: Is there a reason to call this page customization ? Can't we call it Theme instead ?
Later if we want to add more documentation about customization we will create an advanced category with sub documentation pages but from now I think Theme as title will be enough 😄

# Theming

Scaleway UI is based on `@emotion/react` library. It allows you to customize the theme as you need.

> ## Palette

Reviewer: ```suggestion
## Colors
```

I think it might be better to put simple title so if later a user search for colors in documentation search bar he will probably type `colors` and not `palette`. I'm totally open for discussion about that subject

> Here are all colors that we use in Scaleway UI. You can customize it in your custom theme

Reviewer: ```suggestion
Here are all colors that we use in Scaleway UI. You can customize them in your custom theme.
```

        </Box>
      ),
  )}
</div>

## Custom theme

> Here are an exemple custom theme you can use to extend and replace colors from Scaleway UI's [default theme](https://github.com/scaleway/scaleway-ui/blob/master/src/theme/index.js)

Reviewer: ```suggestion
Here is an example of custom theme you can use to extend and replace colors from Scaleway UI's [default theme](https://github.com/scaleway/scaleway-ui/blob/master/src/theme/index.js)
```
